_pages/en/strategy-product-management.md

```diff
@@ -0,0 +1,475 @@
+   - This Business Case is _NOT_ your typical Business Case used to fund a project.
+2. The LOB ADM, CIO, and CFO get into a Memorandum Of Understanding (MOU) by identifying:
+   - source(s) of funds
+   - [level of autonomy](#appendix-c---level-of-autonomy)
+   - expected budget
+   - the [Product's Architectural Boundaries](#appendix-a---product-architectural-boundaries)
+   - the Product's Value Stream
+   - the Product's internal governance structure
+   - the time horizon sought for the blanket approval of priority and funding (in years)
+3. The Major Projects and Investment Board (MPIB)[^6] oversees investment arrangements and progress reporting
+   - Includes TB submissions, review of product status, and reporting requirements
+
+### Product Roadmap as the Investment Plan
+
+The Product's roadmap acts as the Investment Plan for the time horizon and is the basis for accountability.
+
+<div style="display: block; width: inherit; text-align: center; line-height: 3;">
+    <a href="/ITStrategy/assets/images/paa-products-value-stream.png" style="display: block">
+        <img src = "{{ site.baseurl }}/assets/images/product-roadmap.png"
+        alt = "alt text here"
+        />
+    </a>
+    <small>Figure 2. The Product's Roadmap as the Investment Plan.</small>
+</div>
+
+Business Outcomes are mini-business cases (typically 1 pager) and the definition of success for that roadmap step.
+Business Outcome deliveries also act as decision gates in the investment plan.
+
+The blanket approval over the time horizon gives the product's value stream stability.
+However, funding may be augmented based on the desired Business Outcome in the roadmap, at which point departmental governance needs to be engaged.
+This funding is not based on historical spending levels but on future need.
+
+The Roadmap **can and will be re-prioritized**.
+New business outcomes may be added or shifted on its timeline due to experience gained with the product and to changes in departmental or portfolio priorities.
+
+The Roadmap is **reality** for the product team.
+Parallel projects feed the Product Roadmap's backlog and their schedules are expected to be adjusted based on the changes in the product's roadmap.
+This is because it is unrealistic for a product team to manage multiple commitments (e.g., Windows server upgrade, WLM, Accessibility Compliance Project, Technical Debt Programme, one or two Business Enhancements, Benefits Delivery Modernization, etc.).
+Instead, those commitments are part of the Product's backlog and will be dealt with as part of the Product's roadmap.
+Should reality changes (e.g., new priority affecting a product's roadmap horizon), parallel projects absorb the change and adjust their schedules accordingly.
+Should those projects be sources of funds for a product, they are provided only when the Product business outcome plan includes requirements of such projects.
```

Comment:
I don't understand this sentence, maybe that's just me though.

Reply:
Often, a project will span multiple products (e.g. a11y compliance targetting more than one software solution / product). Because it's a project, it seeks commitments for date/time (e.g. Product A, you agree to make the changes in FY 22/23). Because it's the project that gets additional funds (e.g. funds to hire consultants, pay vendor, or just salary dollars) then the product is dependent on that project commitment.

But that's just too far away of commitment. A product doesn't have just a11y compliance but other stuff as well. So instead, it fills its backlog with requirements. The commitment to the a11y project is that when their product roadmap can commit to a release, then the a11y project can release funds to help them.